index.html

```diff
@@ -16,8 +16,8 @@
       <label for="partner_name">Partner name:</label>
       <input type="text" id="partner_name" name="partner_name" />
 
-      <label for="encoded_password">Password:</label>
-      <input type="text" id="encoded_password" name="encoded_password" />
+      <label for="decoded_password">Password:</label>
```

Comment:
I am not sure why the fields here should be encoded or decoded. Am I missing something here? Typically, in the day to day life application, we enter our password not by encoding with some algo. 
Also, in the HTML, the type for the password is not text, rather "password" https://www.w3schools.com/tags/att_input_type_password.asp

Reply:
I just wanted to clarify it is the password before encoding it - this is why I wrote decoded.
I'll change password type, thanks!